src/service/service.py

```diff
@@ -98,29 +99,50 @@ async def invoke(user_input: UserInput) -> ChatMessage:
         raise HTTPException(status_code=500, detail=str(e))
 
 
-async def message_generator(user_input: StreamInput) -> AsyncGenerator[str, None]:
+@app.post("/invoke/research-assistant")
```

Comment:
research-assistant should not be in spread across the service code, only in the setup code (lifespan).

Reply:
Done, removed multiagent implementation